### ISSUE TITLE
How do I use the ladder? Pressing 1 on a laptop does nothing

### ISSUE THREAD

**Issue by sebastien-modley** [Louis-Tarvin/game-off-2023#1]

**Comment by Louis-Tarvin** [comment]
Yeah, I didn't make this super clear (other play-testers were also struggling with this). I've tweaked the levels slightly to better tutorialise things, and also added some more guidance to the itch.io description. Let me know if that makes things clearer!
